### PR TITLE
Install custom cert (if used).

### PR DESCRIPTION
**Pull request by divolgin**
Add the custom cert to `/etc/docker/certs.d` so Docker can pull images from Replicated on-prem registry.

### CODE REVIEW

File: install_scripts/templates/kubernetes/node-join.sh [hunk 1]
```diff
@@ -264,6 +267,11 @@ if [ "$AIRGAP" = "1" ]; then
         mkdir -p "/etc/docker/certs.d/$DAEMON_REGISTRY_ADDRESS"
         promptForCA
         echo "$(echo "$CA" | base64 --decode)" > "/etc/docker/certs.d/$DAEMON_REGISTRY_ADDRESS/ca.crt"
+
+        promptForCert
```
Comment (emosbaugh) [inline install_scripts/templates/kubernetes/node-join.sh]: is a cert always required? if not can you make a flag to skip this prompt and maybe even just make it an argument and not prompt? this is gonna mess with automation

Comment (divolgin) [reply install_scripts/templates/kubernetes/node-join.sh]: good catch.  I'll just remove the prompt

File: install_scripts/app.py [hunk 2]
```diff
@@ -369,6 +370,7 @@ def get_replicated_compose_v3_template_args(replicated_channel=None,
         # query params
         airgap=airgap,
         ca=ca,
+        cert=cert,
```
Comment (emosbaugh) [inline install_scripts/app.py]: this seems silly but i see youre just following ca here

Comment (divolgin) [reply install_scripts/app.py]: yep

**Pull request merged**